### PR TITLE
prep for GA arm docker

### PR DESCRIPTION
**Pull request by sebastian-lerner**
prep for GA

### CODE REVIEW

File: jekyll/_cci2/configuration-reference.md [hunk 1]
```diff
@@ -465,8 +465,8 @@ The `orbs` key is supported in `version: 2.1` configuration
 Key | Required | Type | Description
 ----|-----------|------|------------
 orbs | N | Map | A map of user-selected names to either: orb references (strings) or orb definitions (maps). Orb definitions must be the orb-relevant subset of 2.1 config. See the [Creating Orbs]({{ site.baseurl }}/creating-orbs/) documentation for details.
-executors | N | Map | A map of strings to executor definitions. See the [Executors]({{ site.baseurl }}/configuration-reference/#executors-requires-version-21) section below.
-commands | N | Map | A map of command names to command definitions. See the [Commands]({{ site.baseurl }}/configuration-reference/#commands-requires-version-21) section below.
+executors | N | Map | A map of strings to executor definitions. See the [Executors]({{ site.baseurl }}/configuration-reference/#executors) section below.
```
Comment (rosieyohannan) [inline jekyll/_cci2/configuration-reference.md:468]: Not sure why this is in here, I thought this change was in master. Either way it's correct!

File: jekyll/_cci2/configuration-reference.md [hunk 2]
```diff
@@ -845,8 +845,12 @@ If you are using CircleCI server, contact your system administrator for details
 
 When using the Linux [GPU executor]({{ site.baseurl }}/using-gpu), the available images are:
 
-* `linux-cuda-11:default` v11.4, v11.6, v11.8 (default), Docker v20.10.24
-* `linux-cuda-12:default` v12.0, v12.1 (default), Docker v20.10.24
+* `ubuntu-2004-cuda-11.4:202110-01` - CUDA v11.4.2, Docker v20.10.7, nvidia-container-toolkit v1.5.1-1
```
Comment (rosieyohannan) [inline jekyll/_cci2/configuration-reference.md:848]: Something odd going on here @kbatuigas - these changes were in master and now they are not. Must be something that crept back during the GH apps release I think. Do you know who worked on these changes originally so we can check the content is correct?

Comment (rosieyohannan) [reply jekyll/_cci2/configuration-reference.md:848]: Oh @JalexChen could you check? Looks like this branch somehow is trying to revert your recent changes. Can you confirm the text in the red here is actually correct and what we want?

File: jekyll/_cci2/configuration-reference.md [hunk 3]
```diff
@@ -845,8 +845,12 @@ If you are using CircleCI server, contact your system administrator for details
...
+* `ubuntu-2004-cuda-11.2:202103-01` - CUDA v11.2.1, Docker v20.10.5, nvidia-container-toolkit v1.4.2-1
+* `ubuntu-1604-cuda-11.1:202012-01` - CUDA v11.1, Docker v19.03.13, nvidia-container-toolkit v1.4.0-1
+* `ubuntu-1604-cuda-10.2:202012-01` - CUDA v10.2, Docker v19.03.13, nvidia-container-toolkit v1.3.0-1
+* `ubuntu-1604-cuda-10.1:201909-23` - CUDA v10.1, Docker v19.03.0-ce, nvidia-docker v2.2.2
+* `ubuntu-1604-cuda-9.2:201909-23` - CUDA v9.2, Docker v19.03.0-ce, nvidia-docker v2.2.2
```
Comment (rosieyohannan) [inline jekyll/_cci2/configuration-reference.md:853]:
```suggestion
* `linux-cuda-11:default` v11.4, v11.6, v11.8 (default), Docker v20.10.24
* `linux-cuda-12:default` v12.0, v12.1 (default), Docker v20.10.24
```

File: jekyll/_cci2/configuration-reference.md [hunk 4]
```diff
@@ -1130,8 +1134,8 @@ version: 2.1
 jobs:
   build:
     machine:
-      image: linux-cuda-12:default
-      resource_class: gpu.nvidia.medium
+      image: ubuntu-1604-cuda-10.1:201909-23
+    resource_class: gpu.nvidia.small
```
Comment (rosieyohannan) [inline jekyll/_cci2/configuration-reference.md:1138]:
```suggestion
      image: linux-cuda-12:default
    resource_class: gpu.nvidia.mediuml
```

**Review by rosieyohannan — APPROVED** [review]

File: jekyll/_cci2/building-docker-images.md
Comment (sebastian-lerner) [inline jekyll/_cci2/building-docker-images.md:1]: for pricing, we can point to https://circleci.com/product/features/resource-classes/

File: jekyll/_cci2/building-docker-images.md [hunk 5]
```diff
@@ -82,7 +82,7 @@ Below is a break down of what is happening during this build’s execution:
 ### Resource classes
 {: #resource-classes}
 
-**Arm on Docker** Support for Arm architecture in the Docker execution environment is in **Preview**. For pricing information, and a list of CircleCI Docker convenience images that support Arm resource classes, see the [Arm and Docker Discuss post](https://discuss.circleci.com/t/product-launch-arm-docker-preview/48601).
+**Arm on Docker** For pricing information, and a list of CircleCI Docker convenience images that support Arm resource classes, see the [Arm and Docker Discuss post](https://discuss.circleci.com/t/product-launch-arm-docker-preview/48601).
```
Comment (sebastian-lerner) [inline jekyll/_cci2/building-docker-images.md:85]: for pricing we can point to https://circleci.com/product/features/resource-classes/

Comment (rosieyohannan) [inline jekyll/_cci2/building-docker-images.md:85]:
```suggestion
**Arm on Docker** For pricing information, and a list of CircleCI Docker convenience images that support Arm resource classes, see the [Resource classes page](https://circleci.com/product/features/resource-classes/).
```

File: jekyll/_cci2/configuration-reference.md [hunk 6]
```diff
@@ -1000,7 +1000,7 @@ jobs:
 ###### Arm
 {: #arm }
 
-**Arm on Docker** Support for Arm architecture in the Docker execution environment is in **Preview**. For pricing information, and a list of CircleCI Docker convenience images that support Arm resource classes, see the [Arm and Docker Discuss post](https://discuss.circleci.com/t/product-launch-arm-docker-preview/48601).
+**Arm on Docker** For pricing information, and a list of CircleCI Docker convenience images that support Arm resource classes, see the [Arm and Docker Discuss post](https://discuss.circleci.com/t/product-launch-arm-docker-preview/48601).
```
Comment (rosieyohannan) [inline jekyll/_cci2/configuration-reference.md:1003]:
```suggestion
**Arm on Docker** For pricing information, and a list of CircleCI Docker convenience images that support Arm resource classes, see the [Resource classes page](https://circleci.com/product/features/resource-classes/).
```

**Pull request merged**